src/Action/ExceptionAction.php

```diff
@@ -50,6 +52,10 @@ public function __construct(SerializerInterface $serializer, array $errorFormats
      */
     public function __invoke($exception, Request $request): Response
     {
+        if ($exception instanceof \Throwable) {
+            $exception = FlattenException::createFromThrowable($exception);
+        }
```

Comment:
We have an issue here but I'm not sure where to fix it (it should definitely be handled elsewhere). 
In case of a AccessDeniedException we receive an AccessDeniedHttpException here instead of a FlattenException. Something is not right.

Reply:
@chalasr Thoughts?

Reply:
We started to make `FlattenException`  go away in https://github.com/symfony/symfony/pull/34309, some `\Throwable` aren't flattened anymore at this stage. Handling any `Throwable` seems right, that's what the symfony's `ErrorController` does (which should make this one obsolete btw, we may want to deprecate it on master).

Reply:
Sorry, I don't understand what's the connection between https://github.com/symfony/symfony/pull/34309 and this issue.

The problem here is that we're expecting `FlattenException` but getting `AccessDeniedHttpException` instead...

Reply:
Bad reference sorry. The new ErrorListener does not flatten it anymore unless the $exception argument has a FlattenException type declaration, see https://github.com/symfony/symfony/pull/34312/files#r344517703

Reply:
See https://github.com/symfony/symfony/pull/34411

Reply:
Thanks! I hope it'll be green now.